library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/Mapping/HuntGroup.HuntGroupAbstract.orm.yml

```diff
@@ -36,13 +36,6 @@ Ivoz\Provider\Domain\Model\HuntGroup\HuntGroupAbstract:
       options:
         unsigned: false
       column: ringAllTimeout
-    nextUserPosition:
-      type: smallint
-      nullable: true
-      options:
-        unsigned: true
-        default: '0'
-      column: nextUserPosition
```

Comment:
We should also remove [this reference](https://github.com/irontec/ivozprovider/blob/bleeding/web/admin/application/configs/klear/HuntGroupsList.yaml#L31)

Reply:
You're right, thanks